osm_fieldwork/CSVDump.py

```diff
@@ -32,7 +32,11 @@
 from osm_fieldwork.osmfile import OsmFile
 from osm_fieldwork.xlsforms import xlsforms_path
 
+#Instantiate logger
+log_level = os.getenv("LOG_LEVEL", default="INFO")
 # set log level for urlib
+logging.getLogger("urllib3").setLevel(log_level)
+log_stream = sys.stderr # default log stream
```

Comment:
Same here, it doesn't set anything

Reply:
The lines can be removed in every file probably

Reply:
I used log_level and log_stream to set the default values for logging.basicConfig. They change when verbose is set.

Reply:
The thing is the variables are not used unless verbose is set and basicLogger is set.

In which case the variables are overriden. So the original values defined would never be used.

There is no need to set a default log level in the file, as this will be set by upstream tools that instantiate a logger and then hook into osm-fieldworks logs (with their own log level set).

The verbose flag should only be configurable via the CLI, in which case osm-fieldwork is run standalone and basicLogger should be set.

Reply:
Thank you. I initially did the hardcoding but it looked more verbose. But its less complex that way.
Will change it now.